specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -612,131 +681,545 @@
+          "type": "string"
+        },
+        "customerDisplayName": {
+          "description": "The name of the customer for whom the subscription was created. The field is applicable only for Microsoft Partner Agreement billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
         "displayName": {
           "description": "The name of the subscription.",
+          "type": "string"
+        },
+        "enrollmentAccountId": {
+          "description": "The enrollment Account ID associated with the subscription. This field is available only for the Enterprise Agreement Type billing accounts.",
           "type": "string",
           "readOnly": true
         },
-        "subscriptionId": {
-          "description": "The ID of the subscription.",
+        "enrollmentAccountDisplayName": {
+          "description": "The enrollment Account name associated with the subscription. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "enrollmentAccountSubscriptionDetails": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/EnrollmentAccountSubscriptionDetails",
+          "title": "Enrollment Account Subscription details. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "readOnly": true
+        },
+        "invoiceSectionId": {
+          "description": "The ID of the invoice section to which the subscription is billed. The field is applicable only for Microsoft Partner Agreement billing accounts.",
+          "type": "string"
+        },
+        "invoiceSectionDisplayName": {
+          "description": "The name of the invoice section to which the subscription is billed. The field is applicable only for Microsoft Partner Agreement billing accounts.",
           "type": "string",
-          "format": "uuid",
           "readOnly": true
         },
-        "subscriptionBillingStatus": {
-          "description": "The current billing status of the subscription.",
+        "invoiceSectionName": {
+          "description": "The name of the invoice section to which the subscription is billed. The field is applicable only for Microsoft Partner Agreement billing accounts.",
           "type": "string",
-          "enum": [
-            "Active",
-            "Inactive",
-            "Abandoned",
-            "Deleted",
-            "Warning"
-          ],
-          "x-ms-enum": {
-            "name": "BillingSubscriptionStatusType",
-            "modelAsString": true
-          }
+          "readOnly": true
         },
         "lastMonthCharges": {
-          "description": "The last month charges.",
+          "description": "The last month charges. This field is only available for consumption subscriptions of Microsoft Customer Agreement billing accounts.",
```

Comment:
The last month's charges. This field is only available for usage based subscriptions of Microsoft Customer Agreement billing accounts.